prisma/migrations/20240922092246_neovim_config_unique_for_id_user_id/migration.sql

```diff
@@ -0,0 +1,12 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `markedForDeletion` on the `NeovimConfig` table. All the data in the column will be lost.
+  - A unique constraint covering the columns `[id,userId]` on the table `NeovimConfig` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- AlterTable
+ALTER TABLE "NeovimConfig" DROP COLUMN "markedForDeletion";
```

Comment:
❓ Can we remove any mention of markedForDeletion in this migration as well?

Reply:
Done.